ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-release-4.20.yaml

```diff
@@ -598,6 +598,117 @@ tests:
     test:
     - ref: windows-e2e-operator-test
     workflow: ipi-aws-ovn-hybrid
+- as: e2e-aws-mco-disruptive
+  interval: 48h
```

Comment:
Since you are looking to create these jobs before you have any tests, how about reducing the frequency this runs until you get the tests added.  Unless you expect a quick turnaround there.